### ISSUE TITLE
[BUG] Dash async causes syntax error in IE11

### ISSUE THREAD

**Issue by shammamah** [plotly/dash#1003]
Thank you so much for helping improve the quality of Dash!

We do our best to catch bugs during the release process, but we rely on your help to find the ones that slip through.

**Note - found while working on https://github.com/plotly/dash-core-components/issues/627** 

**Describe your context**
Please provide us your environment so we can easily reproduce the issue.

-  replace the result of `pip list | grep dash` below
```
dash                            1.6.0    
dash-core-components            1.5.0    
dash-dangerously-set-inner-html 0.0.2    
dash-flow-example               0.0.5    
dash-html-components            1.0.1    
dash-renderer                   1.2.0    
dash-table                      4.5.0    
```
-  if frontend related, tell us your Browser, Version and OS

    - OS: Windows
    - Browser: Internet Explorer
    - Version: 11

**Describe the bug**

Using the `dash-core-components` library installed with the latest version of Dash that includes async (`1.6.0`) in an app causes the app to be stuck on "Loading..." and the following errors to show up in the console: 

![screen_shot_2019-11-06_at_1 29 43_pm](https://user-images.githubusercontent.com/15622051/68332696-3e1e5300-00a5-11ea-9b7b-6b95fbdd4177.png)

This is due to an arrow function: 

![Screen Shot 2019-11-06 at 2 51 05 PM](https://user-images.githubusercontent.com/15622051/68332474-e67fe780-00a4-11ea-8692-9362591e89e1.png)

**Expected behavior**

The components should render. 

**Screenshots**

See above.

**Comment by Marc-Andre-Rivet** [comment]
Caused by `/node_modules/`, `^[@]plotly\/.*/` should be included as the non-transpiled source is used directly for the plugins.

This affects dash-renderer, html, dcc, table, bio, daq in production.
This affects the in-review changes:
https://github.com/plotly/dash-cytoscape/pull/75
https://github.com/plotly/dash-canvas/pull/29

**Comment by Marc-Andre-Rivet** [comment]
Also, there are other issues unrelated with async that are breaking IE11. Details in #1006

**Comment by Marc-Andre-Rivet** [comment]
Closed by #1006
